src/tools/interop/idt/README.md

```diff
@@ -0,0 +1,564 @@
+   RPi:
+    ```
+    scp -r ./idt/* kali@kali:/home/kali/idt
+    ```
+1. `ssh` to the RPi, e.g. with `MobaXterm` (the default password for Kali is
+   `kali`)
+    - **_NOTE:_** You may need to wait a few minutes after boot for the `ssh`
+      server to be available on the RPi. Retry if needed!
+    - Use `kali@kali-raspberry-pi` or `kali@$ip` where `$ip` is the RPi's IP
+      found in your router admin panel.
+
+### Configure the RPi
+
+1. Once in an ssh session, set a strong password.
+    ```
+    sudo passwd kali
+    ```
+1. Inside the RPi, configure Wi-Fi. Disconnect ethernet, reboot and connect
+   again.
+    ```
+    nmtui
+    ```
+1. Set up `idt`:
+    ```
+    cd ~                               # Go to idt parent dir
+    source idt/scripts/setup_shell.sh  # Setup atuo aliases
+    source idt/scripts/alias.sh        # Get aliases now
+    idt_bootstrap                      # Initial configuration
+    idt_build                          # Build the container image
+    ```
+
+### Install updates
+
+SCP may not overwrite all files. To clear the `idt` dir off of the RPi safely
+between pushes, exit the container and:
+
+```
+idt_clean
+```
+
+NOTE the idt artifacts directory is contained in idt, so running this will
+delete any artifacts ([TODO] change).
+
+Then from the admin computer:
+
+```
+idt_push
+```
+
+## Debian installation
+
+This package should also work on most other Debian (/based) systems.
+
+The machine running `idt` should be connected to the same Wi-Fi network used for
+testing.
+
+-   From the parent directory of `idt`, run `source idt/scripts/alias.sh`.
+-   Optionally, run `source idt/scripts/setup_shell.sh` to install aliases
+    permanently.
+
+> You may use `idt` natively (Python virtual environment) OR via Docker.
+
+### Python virtual environment
+
+-   `idt` is currently tested on `Python 3.11`.
+-   `adb` and `tcpdump` are required.
+
+Setup a virtual environment and install python dependencies:
+
+```
+idt_go
+cd idt
+python -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
+
+Remember to re-enter the virtual environment as needed. [TODO] script
+
+### Docker
+
+-   Run `idt_build` and `idt_activate` to enter the `idt` environment.
+
+[TODO] Podman
+
+## macOS installation
+
+Most features other than BLE should work on macOS.
+
+Follow the Debian installation steps above, but do not use Docker.
+
+[TODO] macOS BLE support
+
+## User guide
+
+> **_IMPORTANT_**  
+> `idt_` commands are shell aliases helpful for administrative commands.  
+> `idt` invokes the `idt` python package.
+
+RPi users, as needed:
+
+-   For users with Windows admin computers, reconnect e.g., using `MobaXterm`
+-   Other users reconnect `ssh` to the RPi (from your admin computer):
+    ```
+    idt_connect
+    ```
+-   Run the `idt` container (from the RPi):
+    ```
+    idt_activate
+    ```
+
+### Capture
+
+```
+idt capture -h
```

Comment:
After doing `idt_connect`, then `idt capture -h` I encounted this...

```bash
┌──(kali㉿kali-raspberry-pi)-[~]
└─$ idt capture -h
Command 'idt' not found, but can be installed with:
sudo apt install ncl-ncarg
Do you want to install it? (N/y)
```

Reply:
In the Pi, idt_activate` is expected before executing `idt ...` commands. 

## User guide

> **_IMPORTANT_**  
> `idt_` commands are shell aliases helpful for administrative commands.  
> `idt` invokes the `idt` python package.

RPi users, as needed:

-   For users with Windows admin computers, reconnect e.g., using `MobaXterm`
-   Other users reconnect `ssh` to the RPi (from your admin computer):
    ```
    idt_connect
    ```
-   Run the `idt` container (from the RPi):
    ```
    idt_activate
    ```

### Capture

Reply:
Resolving this for now, please reopen if needed! I didn't think it was worth reminding users to run activate for every command.